src/extensions/surveys.tsx

```diff
@@ -604,6 +604,14 @@ export function usePopupVisibility(
                 sessionRecordingUrl: posthog.get_session_replay_url?.(),
             })
             localStorage.setItem('lastSeenSurveyDate', new Date().toISOString())
+            setTimeout(() => {
+                const inputField = document
+                    .querySelector('.PostHogWidget' + survey.id)
```

Comment:
lets move PostHogWidget to a const, we are repeating many times over

Reply:
moved to a function as I think it's more clear [5743518](https://github.com/PostHog/posthog-js/pull/1772/commits/57435189533c9b68840ead8ed6bddfa3cd2b0b05)